test-curves/src/bn384_small_two_adicity/tests.rs

```diff
@@ -32,23 +32,17 @@ fn test_fq() {
     }
 }
 
-#[test]
-fn test_g1_projective_curve() {
-    curve_tests::<G1Projective>();
-    sw_tests::<g1::Parameters>();
-}
-
-#[test]
-fn test_g1_projective_group() {
-    let mut rng = ark_std::test_rng();
-    let a: G1Projective = rng.gen();
-    let b: G1Projective = rng.gen();
-    group_test(a, b);
-}
-
 #[test]
 fn test_g1_generator() {
     let generator = G1Affine::prime_subgroup_generator();
     assert!(generator.is_on_curve());
     assert!(generator.is_in_correct_subgroup_assuming_on_curve());
 }
+
+#[test]
+fn test_g1() {
```

Comment:
Do all other curves follow this test format? (`test_g1`, not `test_g1_projective_curve` and `test_g1_projective_group`) If so, PR looks good to merge to me